### PR TITLE
Improve task debugging.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Add an env var so we don't need to comment out code.

### CODE REVIEW

File: src/clusterfuzz/_internal/bot/tasks/commands.py [hunk 1]
```diff
@@ -285,8 +285,8 @@ def process_command_impl(task_name,
     # A misconfiguration led to this point. Clean up the job if necessary.
     # TODO(ochang): Remove the first part of this check once we migrate off the
     # old untrusted worker architecture.
-    # Comment this "if" out to run a task locally.
```
Comment (vitorguidi) [inline src/clusterfuzz/_internal/bot/tasks/commands.py:288]: What situation should I comment this in? I was able to run fuzz task locally without touching this

File: src/clusterfuzz/_internal/bot/tasks/commands.py [hunk 2]
```diff
@@ -285,8 +285,8 @@ def process_command_impl(task_name,
...
-    if (not environment.is_trusted_host(ensure_connected=False) and
+    if (not environment.get_value('DEBUG_TASK') and
```
[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/bot/tasks/commands.py:288]: I think it's for formatting.

File: src/clusterfuzz/_internal/bot/tasks/commands.py (hunk 1 above)
Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/bot/tasks/commands.py:288]: I do remember things hanging when running tasks on my own machine. I'll have to run it without this and find out.

Comment (vitorguidi) [reply src/clusterfuzz/_internal/bot/tasks/commands.py:288]: Yeah, that is indeed true. I take that back

**Review by vitorguidi — APPROVED** [review]
lgtm

**Pull request merged**